astropy/units/core.py

```diff
@@ -235,7 +235,8 @@ def set_enabled_equivalencies(self, equivalencies):
 
         Parameters
         ----------
-        equivalencies : list of equivalent pairs
+        equivalencies : list of tuple
+            list of equivalent pairs
```

Comment:
Here and below, capitalize and combine with below, i.e., "List of equivalent pairs. E.g., as returned by"

Reply:
👍